src/main/java/uk/gov/hmcts/reform/civil/controllers/testingsupport/TestingSupportController.java

```diff
@@ -162,11 +167,38 @@ public ResponseEntity<String> getCaseDismissalScheduler() {
         }
         return new ResponseEntity<>(responseMsg, HttpStatus.OK);
     }
-
+  
     @GetMapping("/testing-support/case/{caseId}")
     public ResponseEntity<CaseData> getCaseData(@PathVariable("caseId") Long caseId) {
 
         CaseData caseData = caseDetailsConverter.toCaseData(coreCaseDataService.getCase(caseId));
         return new ResponseEntity<>(caseData, HttpStatus.OK);
+
+    }
+
+    @GetMapping("/testing-support/{caseId}/trigger-hearing-fee-paid")
+    public ResponseEntity<String> getHearingFeePaidEvent(@PathVariable("caseId") Long caseId) {
+
+        String responseMsg = "success";
+        var event = new HearingFeePaidEvent(caseId);
+        try {
+            hearingFeePaidHandler.moveCaseToPrepareForHearing(event);
+        } catch (Exception e) {
+            responseMsg = "failed";
+        }
+        return new ResponseEntity<>(responseMsg, HttpStatus.OK);
+    }
+
+    @GetMapping("/testing-support/{caseId}/trigger-hearing-fee-unpaid")
+    public ResponseEntity<String> getHearingFeeUnpaidEvent(@PathVariable("caseId") Long caseId) {
+
+        String responseMsg = "success";
+        var event = new HearingFeeUnpaidEvent(caseId);
+        try {
+            hearingFeeUnpaidHandler.moveCaseToStruckOut(event);
+        } catch (Exception e) {
+            responseMsg = "failed";
```

Comment:
Please verify that in case of error the pods logs in preview will show a stack trace of the exception and let me know.
If that's not the case, I will ask that this line be modified.